src/agent/src/config.rs

```diff
@@ -63,7 +62,6 @@ pub struct AgentConfig {
     pub server_addr: String,
     pub unified_cgroup_hierarchy: bool,
     pub tracing: bool,
-    pub supports_seccomp: bool,
```

Comment:
I think you should keep this field, unless it was added by MSFT folk.

Reply:
Does it not make sense to follow the upstream change? https://github.com/microsoft/kata-containers/commit/a131eec5c1517f7120725d287f481e47ce56fdc3

Reply:
I think this field "is never read". I always get this warning when building the agent
<img width="347" alt="image" src="https://github.com/user-attachments/assets/79bb6dc6-8eb3-4e5b-82d7-139c35aff14a" />

Reply:
Yup. This seems to be treated as an error with upcoming versions (either remove or allow this one via linter). This change follows what was changed upstream.